app/domain_models/record_sets_create.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'common_error'
+
+class RecordSetsCreate
+
+    attr_accessor :user_id, :user, :content, :set, :rep, :weight
+
+    def initialize(user_id, content, set, rep, weight)
+        @user_id = user_id
+        @content = content
+        @set = set
+        @rep = rep
+        @weight = weight
+        @user = User.find_by(id: user_id)
+    end
+
+    def create_set_record
+        user_not_found!
+        content_is_wrong!
+        set_is_wrong!
+        rep_is_wrong!
+        weight_is_wrong!
+        succeed_to_create_set
+    end
+
+    private
+
+    def user_not_found!
```

Comment:
I think it is not the knowledge for RecordSetsCreate.
It is more like LoginUser.